[2025-06-27T06:24:19.908+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:24:20.019+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.create_ga4_events_dataset scheduled__2025-03-19T02:10:00+00:00 [queued]>
[2025-06-27T06:24:20.087+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.create_ga4_events_dataset scheduled__2025-03-19T02:10:00+00:00 [queued]>
[2025-06-27T06:24:20.096+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:24:20.400+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-03-19 02:10:00+00:00
[2025-06-27T06:24:20.521+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1460) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:24:20.553+0000] {standard_task_runner.py:72} INFO - Started process 1499 to run task
[2025-06-27T06:24:20.580+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_dev', 'create_ga4_events_dataset', 'scheduled__2025-03-19T02:10:00+00:00', '--job-id', '358', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpr225yy99']
[2025-06-27T06:24:20.612+0000] {standard_task_runner.py:105} INFO - Job 358: Subtask create_ga4_events_dataset
[2025-06-27T06:24:21.023+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_dev.create_ga4_events_dataset scheduled__2025-03-19T02:10:00+00:00 [running]> on host 4e2c61145aac
[2025-06-27T06:24:21.722+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-03-19T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-19T02:10:00+00:00'
[2025-06-27T06:24:22.118+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:24:22.388+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T06:24:22.657+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T06:24:22.670+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T06:24:22.672+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T06:24:22.674+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T06:24:24.097+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T06:24:25.033+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:24:25.067+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-03-19T02:10:00+00:00, execution_date=20250319T021000, start_date=20250627T062420, end_date=20250627T062425
[2025-06-27T06:24:25.272+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:24:25.497+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
